projects/observability/src/shared/components/gauge/gauge-chart.component.ts

```diff
@@ -0,0 +1,99 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, OnChanges, Output } from '@angular/core';
+import { TypedSimpleChanges } from '@hypertrace/common';
+import { maxBy } from 'lodash-es';
+import { HistogramBarData } from '../histogram/histogram-chart.component';
+
+@Component({
+  selector: 'ht-gauge-chart',
+  styleUrls: ['./gauge-chart.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="gauge-chart">
+      <ng-container *ngFor="let item of this.itemOptions">
+        <div class="border-top"></div>
+        <div
+          class="label"
+          [htcTooltip]="item.label"
+          (click)="this.onItemClick(item)"
+          [ngClass]="{ clickable: this.labelClickable }"
+          [ngStyle]="{ color: item.color }"
+        >
+          {{ item.label }}
+        </div>
+        <div class="progress">
+          <div class="progress-value" [ngStyle]="{ width: item.width, backgroundColor: item.color }"></div>
+        </div>
+        <div class="value">
+          <span>{{ item.value | htcDisplayNumber }}</span>
+        </div>
+      </ng-container>
+    </div>
+  `
+})
+export class GaugeChartComponent<T extends GaugeData = GaugeData> implements OnChanges {
```

Comment:
This design is now similar to old top-n and histogram. :)
`https://github.com/hypertrace/hypertrace-ui/blob/main/projects/observability/src/shared/components/histogram/histogram-chart.component.ts`

Reply:
We should name it differently for sure. Gauge chart has a different meaning.